features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -439,6 +466,37 @@ internal class TelemetryEventHandler(
         return RumContext.fromFeatureContext(rumContext)
     }
 
+    private fun computeEffectiveSampleRate(
+        properties: Map<String, Any?>? = null,
+        eventSpecificSamplingRate: Float? = null
+    ): Float {
+        val telemetrySampleRate = rumConfig?.telemetrySampleRate?.percent() ?: return 0f
+
+        val creatingSamplingRate = properties
+            ?.getFloat(InternalTelemetryEvent.CREATION_SAMPLING_RATE_KEY)
+            ?.percent() ?: 1.0
+
+        val reportingSamplingRate = properties
+            ?.getFloat(InternalTelemetryEvent.REPORTING_SAMPLING_RATE_KEY)
+            ?.percent() ?: 1.0
+
+        val eventSamplingRate = eventSpecificSamplingRate?.percent() ?: 1.0
+
+        val sessionSampleRate = rumConfig?.sampleRate?.percent() ?: 1.0
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ why do we report this here ? Is this being used to sample telemetry events ?

Reply:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏ I just double checked in the sdk so `sessionSampleRate` it is not and it should not affect the telemetry event sampling. We need to fix this.

Reply:
yep, removed from computation